Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/blockstack/stacks-blockchain"
 description = "Reference implementation of the Stacks 2.0 Blockchain"
 keywords = [ "stacks", "stx", "bitcoin", "crypto", "blockstack", "decentralized", "dapps", "blockchain" ]
 readme = "README.md"
+resolver = "2"
```

Comment:
Note for reviewers:
This is required in order for the `[target.'cfg(...)] <enable ASM>` directives below to work, see https://doc.rust-lang.org/nightly/cargo/reference/features.html#feature-resolver-version-2

Without this, the `features = ["arm"]` directive is either ignored/disabled for all targets, or enabled for all targets (depending on the specific project config).